rotkehlchen/tests/unit/test_eth2.py

```diff
@@ -501,51 +501,51 @@ def test_validator_daily_stats(price_historian, function_scope_messages_aggregat
         timestamp=1607990400,  # 2020/12/15
         start_usd_price=FVal(1.55),
         end_usd_price=FVal(1.55),
-        pnl=FVal('0.01196'),
+        pnl=FVal('0.0119'),
```

Comment:
Guys I have fixed these in develop. Please, please, please **REBASE** the PR on latest develop when sending it to me for reviews. Especially such a huge beast as this PR.

You are duplicating work like this .... solving the same stuff I have done in smaller PRs in develop. Such a waste ....